frontends/election-manager/src/components/smartcard_modal/program_election_card_view.tsx

```diff
@@ -1,12 +1,23 @@
 import React, { useContext } from 'react';
+import styled from 'styled-components';
 import { assert } from '@votingworks/utils';
-import { Button, Prose } from '@votingworks/ui';
+import { Button, fontSizeTheme, HorizontalRule, Prose } from '@votingworks/ui';
 import { CardProgramming } from '@votingworks/types';
 
 import { AppContext } from '../../contexts/app_context';
+import { electionToDisplayString } from './elections';
 import { generatePin } from './pins';
 import { SmartcardActionStatus, StatusMessage } from './status_message';
 
+interface HeadingProps {
+  marginTop: string;
+}
+
+const Heading = styled.h1<HeadingProps>`
+  /* stylelint-disable-next-line declaration-no-important */
```

Comment:
Necessary to override the high-specificity `Prose` styles - will add a clarifying comment!

Reply:
Why aren't we using the negative-margin on the status message as I had committed?

This implementation is a bit odd as it adds styles to one element when another element is introduced.

Additionally, we should avoid using `!important` as this will likely cause us major headaches in the future.

Reply:
Another solution for the styling of the status message and accounting for the Prose spacing is to place the message in a separate Prose block before the current one. Doing so will eliminate the margin between these elements and then we can simply add the desired margin instead of overriding any margins.

Reply:
Yes good q! I initially switched to a modified top margin on the heading because 1) I generally find positive margins more intuitive than negative ones (don't feel strongly about this, though) and 2) I wanted to be able to remove the margin when `StatusMessage` is displaying an in-progress modal (more on this below).

That said, you raise some really good points about not using `!important` (the need to use it was a red flag for me, too).

Working on some updates! I like the idea of using two separate `Prose` blocks. I'm also looking into better distinguishing status messages for success and error messages from in-progress modals. They're really two separate things, but both captured in the single `StatusMessage` component at the moment. Separating them out will help eliminate challenge 2.

Reply:
https://github.com/votingworks/vxsuite/pull/2267/commits/0c443ac0c24b03350597322b7489abe5144f52bb + https://github.com/votingworks/vxsuite/pull/2267/commits/c986a9beda806953b218d4dd58316d2330d1a70b?w=1